notebooks/Tutorial2_Parameterization.py

```diff
@@ -171,12 +171,12 @@
     label="Avatars",
 )
 
-ax.set_title("Projections of original and avatars produced with large k")
+ax.set_title("Projection of originals and avatars produced with large k")
 # -
 
 # We observe that the area covered by avatars generated with a low *k* is much closer to the area covered by original data points. We can also see that with a low *k*, some avatars are close to original points that are isolated. This may pose a risk of re-identification. This explains the drop in privacy level when reducing *k*.
 #
-# Avatars produced with a large *k* are significantly further away from isolated originals and so ensure their privacy. However care should be taken in setting *k*  with values that are not too high to prevent a drop in utility level. The drop in utility level is represented by the area covered by avatars being much smaller than the ones of originals.
+# Avatars produced with a large *k* are significantly further away from isolated originals and so ensure their privacy. However **care should be taken in setting *k***  with values that are not too high to prevent a drop in utility level. The drop in utility level is represented by the area covered by avatars being much smaller than the ones of originals.
```

Comment:
```suggestion
# Avatars produced with a large *k* are significantly further away from isolated originals and so ensure their privacy. However **care should be taken in setting k**  with values that are not too high to prevent a drop in utility level. The drop in utility level is represented by the area covered by avatars being much smaller than the ones of originals.
```

Reply:
Actually I think it is find like that as the whole sentence is bold and k is italic